PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -905,6 +905,37 @@ int OrdinalIgnoreCaseCompareStrings(LPCTSTR sz1, LPCTSTR sz2)
 	}
 }
 
+std::string CultureSensitiveIgnoreCaseMapString(LPCTSTR s) {
+	int len = wcslen(s);

```

Comment:
The `*len` family of functions all return `size_t` (a dynamic type).
A 64-bit `size_t` is `typedef`'d as `unsigned long long`: https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/46860742/job/c0du01opkdnfbli1/messages